script.py

```diff
@@ -0,0 +1,152 @@
+import subprocess
+import os
+import boto3
+import argparse
+import logging
+from datetime import datetime, timedelta
+from dotenv import load_dotenv
+
+load_dotenv()
+logging.basicConfig(filename="app.log", level=logging.INFO)
+
+
+def mongodump(uri, outpath):
+    command = [
+        "mongodump",
+        "--uri",
+        uri,
+        "--out",
+        outpath,
+    ]
```

Comment:
add the output logs to file instead of terminal
you can add [..., ">>", output_file_path] to the existing code to achieve this